jwst/pipeline/calwebb_spec3.py

```diff
@@ -2,7 +2,7 @@
 
 from ..stpipe import Pipeline
 from .. import datamodels
-from ..exp_to_source import exp_to_source
+from ..exp_to_source import multislit_to_container
 
 # step imports
 from ..skymatch import skymatch_step

```

Comment:
Yes, mrs_imatch_step replaces skymatch_step. You can call it whatever you like for use within the calspec3 module (i.e. the mapping within the step_defs). Not exactly sure what you mean by the user-level step name. If you mean the name that would be used to run the step individually, yes they would need to use either mrs_imatch or MRSIMatchStep. Unless we decide we want to rename it to something a little simpler.